packages/stashjs/src/stash-internal.ts

```diff
@@ -308,15 +308,22 @@ export class StashInternal {
     return await sequence(
       _ => convertAsyncErrorsTo(DecryptionFailure, this.sourceDataCipherSuiteMemo.freshValue()),
       async cipher =>
-        Ok(await Promise.all(res.collections!.map(async info => {
```

Comment:
Changes in this file are from running prettier. I forgot run prettier in https://github.com/cipherstash/cipherstash.js/pull/11.